src/block/index.scss

```diff
@@ -0,0 +1,103 @@
+@import 'stars/star.css';
+@import 'general.css';
+
+.product {
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  width: 200px;
+
+  &__card {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto;
+  }
+
+  &__name {
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: #060b35;
+    margin: 40px auto 0;
+  }
+
+  &__code {
+    margin: 4px 0 0;
+    font-family: Roboto, sans-serif;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: #616070;
+  }
+}
+
+.group {
+  display: flex;
+  justify-content: space-between;
+  &--top {
+    margin-top: 16px;
+  }
+
+  &__reviews {
+    font-family: Roboto, sans-serif;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: right;
+    height: 16px;
+    display: flex;
+    align-items: center;
+  }
+}
+
+.prace {
+  margin-top: 24px;
+  height: 18px;
+  justify-content: space-between;
```

Comment:
it should be the product block, you should have only two blocks on this task (product and stars) and move each block to separate file